app/views/admin/cx_collections/show.html.erb

```diff
@@ -15,6 +15,17 @@
   <% end %>
 </p>
 
+<% if @cx_collection.submitted? %>
```

Comment:
show a success-alert for a `submitted` Collection, showing the time of submission